ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1003,15 +1003,16 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_pause_continue(const mavlink_comma
 
     // requested pause from GCS
     if ((int8_t) packet.param1 == 0) {
-        copter.mode_auto.mission.stop();
-        copter.mode_auto.loiter_start();
+        // copter.mode_auto.mission.stop();
```

Comment:
can you remove these commented out lines?   we normally don't leave commented out lines in the code.